pleskdistup/common/src/dpkg.py

```diff
@@ -122,9 +126,54 @@ def safely_install_packages(
     install_packages(pkgs, repository, force_package_config)
 
 
+def apt_get_retry_temp_fails(
+    apt_get_cmd: typing.List[str],
+    tmpfail_retry_intervals: typing.Optional[typing.List[int]],
+    collect_stdout: bool = False
+) -> str:
+    cant_get_lock = False
+    stdout = []
+
+    if tmpfail_retry_intervals is None:
+        tmpfail_retry_intervals = DPKG_TEMPFAIL_RETRY
+
+    def process_stdout(line: str) -> None:
+        if collect_stdout:
+            nonlocal stdout
+            stdout.append(line)
+        log.info(line)
+
+    def process_stderr(line: str) -> None:
+        log.err(line)
```

Comment:
Questionable that this is an error in terms of log.

Reply:
If the apt process is failing and printing to stderr, for me it's naturally ERR log. However we print in info level that it's OK and will be retried

Reply:
apt-get is not the only process to be launched using these functions.

Reply:
Changed func name if it's enough?